mods/default/nodes.lua

```diff
@@ -151,6 +151,10 @@ default:sign_wall
 default:ladder
 
 default:fence_wood
+default:fence_acacia_wood
+default:fence_junglewood
```

Comment:
For backwards compatibility and simplicity we decided to keep the name unchanged, pine_wood and acacia_wood have the new 2-word form because they are new.